playbooks/library/raxmon.py

```diff
@@ -160,6 +160,21 @@ def main():
     # importing the correct libraries
     from rackspace_monitoring.providers import get_driver
     from rackspace_monitoring.types import Provider
+    import libcloud.security
+
+    # Provide the ability to enable or disable SSL certificate verification
+    # with raxmon
+    CA_CERT_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)),
```

Comment:
This is the __file__ path of the raxmon module. Are we storing, or do we expect to store a cacert here?